### PR TITLE
Fix/drawer mask not disappear

### PR DESCRIPTION
**Pull request by zxyao145**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
#3046

### 📝 Changelog

> **Note:** this pr does not synchronize the implementation of antd-react, because using animation in the blazer is not very friendly

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    fix: drawer mask not disappear       |
| 🇨🇳 Chinese |     修复 drawe mask 不会消失的问题      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by ElderJames** [comment]
Thanks for contribution @zxyao145 , please fix the test.

**Comment by zxyao145** [comment]
@ElderJames The unit test of drawer component has been fixed, and you may need to repair the test project

File: AntDesign.lutconfig [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<LUTConfig Version="1.0">
+  <Repository />
+  <ParallelBuilds>true</ParallelBuilds>
+  <ParallelTestRuns>true</ParallelTestRuns>
+  <TestCaseTimeout>180000</TestCaseTimeout>
+</LUTConfig>
```
Comment (ElderJames) [inline AntDesign.lutconfig:6]: What is this?

Comment (zxyao145) [reply AntDesign.lutconfig:6]: Live unit testing config. 
Sorry, this is generated automatically by vs, and  submitted through `git add .`. I'll fix it this noon.

**Review by ElderJames — APPROVED** [review]

**Pull request merged**